sssekai/abcache/__init__.py

```diff
@@ -444,7 +444,13 @@ def request_packed(self, method: str, url: str, data: dict = None, **kwargs):
             data = packb(data)
             data = encrypt(data, SEKAI_APIMANAGER_KEYSETS[self.config.app_region])
         resp = self.request(method=method, url=url, data=data, **kwargs)
-        resp.raise_for_status()
+        if 400 <= resp.status_code < 600:
+            try: # print the error message provided by the API.
+                self.response_to_dict(resp)
+                print(resp)
```

Comment:
Please use `logger.warn("%s" % resp)` for error reporting facilities since that's how the rest of AbCache did things